### PR TITLE
Add unit tests for app.py

### PR DESCRIPTION
**Pull request by cbonoz**

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=441) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=441) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=441) it.</sub>

File: test/test_app.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+from unittest.mock import MagicMock
```
Comment (mbernier) [inline test/test_app.py]: For Python 2.6, 2.7 builds to pass, we have to add a little bit of trickery:

```
try:
    import unittest2 as unittest
except ImportError:
    import unittest
```

and for mock, it looks like we can borrow some advice from [this guy](http://railslide.io/travis-mock-afternoon-of-frustration.html):
```
try:
    from mock import patch, mock_open
except ImportError:
    from unittest.mock import patch, mock_open
```

Can you please make these two changes? I am hoping the build will pass once this is done!

**Comment by mbernier** [comment]
@cbonoz I am checking in to see if you can please make the suggested change?

**Comment by cbuonocore-toasttab** [comment]
Go ahead and close this.

**Comment by cbonoz** [comment]
@thinkingserious 👍 Always appreciate the swag. Thanks guys.

**Comment by thinkingserious** [comment]
Hi @cbonoz,

Why did you close this PR?

With Best Regards,

Elmer

**Comment by cbonoz** [comment]
Coverage was added by https://github.com/sendgrid/sendgrid-python/pull/433

**Comment by thinkingserious** [comment]
Cool, thanks for the update :)

**Comment by mbernier** [comment]
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/30min) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request closed without merging**